config/vufind/sitemap.ini

```diff
@@ -43,13 +43,37 @@ fileLocation   = /tmp
 ;    default will be "Solr,/Record/"
 index[] = "Solr,/Record/"
 ;index[] = "SolrAuth,/Authority/Record?id="
+;   Indexes can be completely disabled by setting this to false:
+;index = false
 
 ;  This setting controls how IDs are retrieved from the index. It may be
 ;  'search' (the default, most compatible but slower method), or 'terms' (the
 ;    faster option). Note that 'terms' method does not support hidden filters or
 ;    other limiting options and requires that the index has terms enabled.
 retrievalMode = search
 
+;  This settings controls whether different language versions are added to the
+;  sitemap. Default is false.
+;  Note that including language versions will make the sitemap files substantially
+;  larger and may not be very useful if language detection is enabled in VuFind.
+indexLanguageVersions = false
```

Comment:
Changing this to true is having no effect on the main record sitemaps generated by my test instance, whether or not I also set allowedLanguages below. It does work for content pages, though. Is this intentional?

Reply:
Oops. I forgot to update the relevant part when changing the language handling mechanism. Should work now.

Reply:
Hmm, now I always get the alternate languages, regardless of how I set indexLanguageVersions.

Reply:
Messing around with this makes me wonder whether we really need indexLanguageVersions AND allowedLanguages. Could we just have a single setting that is true for all, false for none, and an array for selected values? (I'm fine with keeping it as it is when you prefer... but I'm wondering if having multiple settings is making it easier to introduce bugs).

Reply:
Fix the setting (another casualty of bad refactoring), but I think you're right. I'll refactor a bit more!

Reply:
I'm having a bit of problem detecting the type of the setting. Do we have such settings somewhere else? Or should it be empty for none, `*` for all and a comma-separated list for specific ones (like e.g. `orFacets` in facets.ini)?

Reply:
Yes, I like that idea! Locale names should never contain commas, so there's no technical need to format them as an array in the .ini file anyway.

Reply:
Should be done now. I also moved the x-default logic from Sitemap to Generator to keep the logic out of Sitemap. And fixed a mistake in fallback language handling to avoid duplicate hreflang entries.